### PR TITLE
First message

### PR DESCRIPTION
**Pull request by pvseydlitz**

### CODE REVIEW

File: src/Message.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import placeholder from './Placeholder.svg'
+const messages = require('./messages.json')
```
Comment (dvtdyn) [inline src/Message.js]: use import messageData from './messages.json'

File: src/Message.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  const Wrapper = styled.div`
+    display: grid;
+    grid-template-columns: auto 100px;
+  `
+  const Description = styled.p`
+    margin: 0;
+    font-size: 16px;
+    color: rgb(107, 107, 107);
+  `
+  const Picture = styled.img`
```
Comment (dvtdyn) [inline src/Message.js]: maybe use grid

File: src/Message.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    position: absolute;
+    right: 22px;
+    bottom: 27px;
+  `
+
+  return messages.map((message, index) => (
```
Comment (dvtdyn) [inline src/Message.js]: Maybe focus in message only on one message and use map in app

File: public/index.html [hunk 4]
```diff
@@ -12,7 +12,8 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```
Comment (dvtdyn) [inline public/index.html]: Task research manifest

File: src/Message.js (hunk 1 above)
Comment (pvseydlitz) [reply src/Message.js]: done

File: src/Message.js (hunk 2 above)
Comment (pvseydlitz) [reply src/Message.js]: created task for next sprint

**Comment by dvtdyn** [comment]
Well done Philipp

**Pull request merged**